ranch/storage.go

```diff
@@ -314,7 +329,11 @@ func (s *Storage) updateDynamicResources(lifecycle *crds.DRLCObject, resources [
 			toDelete = append(toDelete, res)
 		}
 	}
-	logrus.Infof("DRLC type %s: adding %+v, deleting %+v", lifecycle.Name, toAdd, toDelete)
+	logrus.WithFields(logrus.Fields{
+		"type":               lifecycle.Name,
+		"add_items_count":    len(toAdd),
+		"delete_items_count": len(toDelete),
+	}).Info("Updating DLRCs")
```

Comment:
nit: DRLCs, not DLRCs :)

Reply:
I actually removed that log completely because its redundant with the log that logs each create/delete individually